osu.Game/Screens/Edit/Compose/Components/Timeline/TimelineHitObjectBlueprint.cs

```diff
@@ -122,6 +125,9 @@ protected override void LoadComplete()
                     comboIndexBindable = comboInfo.ComboIndexBindable.GetBoundCopy();
                     comboIndexBindable.BindValueChanged(_ => updateColour(), true);
 
+                    comboIndexWithOffsetsBindable = comboInfo.ComboIndexWithOffsetsBindable.GetBoundCopy();
+                    comboIndexWithOffsetsBindable.BindValueChanged(_ => updateColour(), true);
```

Comment:
You could probably only leave the `true` arg on the last call and trim the one above, but that's a rather minor point. (This happens in more places than just this one in this diff.)

Reply:
No harm in doing that.